blog/2015/03/first-software-carpentry-korea.html

```diff
@@ -0,0 +1,78 @@
+---
+layout: blog
+root: ../../..
+author: Victor (Kwnagchun) Lee
+title: "The First Software Carpentry in Korea"
+date: 2015-03-02
+time: "23:37:00"
+category: ["Workshops", "Korea", "Seoul", "Rapa", "xwMOOC"]
+---
+<!-- start excerpt -->
+<p> Last week, xwMOOC <a href="http://www.xwmooc.net"></a> ran the first <a href="http://statkclee.github.io/2015-02-25-seoul/">Software Carpentry workshop</a> in Seoul. The workshop delivered not only an introduction to Unix Shell, Git and GitHub, Python and SQL but also computer science unplugged, rur-ple, python for informatics and cloud basics. Listening to attendees feedback, attendees concerns such as software business, all about what start-ups should know, how software business environment looks like and so on are provided during three days workshops.
+</p>
+<!-- end excerpt -->
+
+<p> Due to lack of instructors, Victor did teach all the lessons. Unfortunately, Dave, who is supposed to be a helper, could attend the first workshop such that one of attendees, Mr. Seo, became a volunteer and did a wonderful job. Even though there were some software installation problems, Mr. Seo and Victor could overcome some technical difficulties. </p>
+
+<p>Initially we want to start small, but we had some concerns about churn rates. Therefore, we initially limit the number of attendees up to 23. Because the first software carpentry workshop was free, the registration has been closed two weeks ago. The final attendees have been check by <a href="http://www.rapa.or.kr/">RAPA(host)</a> and the number was 17. Because the lesson contents of software carpentry are quite new, sort of advanced to Korean, and tailered to science and engineering, from the beginning Victor and RAPA paid special attentions to retention rates. Finally, we kept 12 attendees, which is over 70%. Our retention goal was 50%, 8 attendees.   
```

Comment:
- Software Carpentry must have first letter capitalized.

~~~
<p>Initially we want to start small, but we had some concerns about churn rates. Therefore, we initially limit the number of attendees up to 23. Because the first Software Carpentry workshop was free, the registration has been closed two weeks ago. The final attendees have been check by <a href="http://www.rapa.or.kr/">RAPA(host)</a> and the number was 17. Because the lesson contents of Software Carpentry are quite new, sort of advanced to Korean, and tailered to science and engineering, from the beginning Victor and RAPA paid special attentions to retention rates. Finally, we kept 12 attendees, which is over 70%. Our retention goal was 50%, 8 attendees.
~~~